tests/common/utilities.py

```diff
@@ -467,6 +467,14 @@ def get_test_server_visible_vars(inv_files, server):
     return vm.get_vars(host=test_server_host)
 
 
+def is_ip_reachable(duthost, ip_address):
```

Comment:
get_dut_current_passwd, the ssh session is from test agent to DUT, in other words sonic-mgmt-container -> DUT.
But the detection here is from DUT -> DUT. 
Which is not 100% aligned with the get_dut_current_passwd logic.